### PR TITLE
Ubuntu/bionic

### PR DESCRIPTION
**Pull request by TheRealFalcon**

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -31,14 +33,16 @@ cloud-init (22.2-145-gaa59209a-0ubuntu1~18.04.1) UNRELEASED; urgency=medium
     + tests: can run without azure-cli, tests expect inactive ansible
       (#1643)
     + typing: Type UrlResponse.contents (#1633) [Alberto Contreras]
-    + testing: fix references to `DEPRECATED.` (#1641) [Alberto Contreras]
+    + testing: fix references to `DEPRECATED.` (#1641) [Alberto
```
Comment (blackboxsw) [inline debian/changelog:36]: I don't like that our current tooling with gbp is separating Alberto Contreras across two lines, can you manually bump his authorship name down to the next line in these few cases where it got separated.

Comment (TheRealFalcon) [reply debian/changelog]: Done (and for the other branches as well).

**Review by blackboxsw — CHANGES_REQUESTED** [review]
our new-upstream-snapshot tooling isn't behaving with line-wrapping or ordering of new commits in chronological order. Here is a manual diff what changes I suggest:
```diff
diff --git a/debian/changelog b/debian/changelog
index c79b0b35..f9f23322 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -3,7 +3,6 @@ cloud-init (22.3-3-g9f0efc47-0ubuntu1~18.04.1) bionic; urgency=medium
   * d/control: add python3-debconf to Depends and Build-Depends
   * d/cloud-init.postinst: lintian fixes:
     + Fix command-with-path-in-maintainer-script for grub-install
-  * d/p/renderer-do-not-prefer-netplan refresh to activators change
   * d/p/expire-on-hashed-users.patch:
     Add patch to ensure password expire doesn't apply to hashed users
   * d/source/lintian-overrides: lintian fixes:
@@ -13,8 +12,23 @@ cloud-init (22.3-3-g9f0efc47-0ubuntu1~18.04.1) bionic; urgency=medium
     + debian/patches/ec2-dont-apply-full-imds-network-config.patch
     + debian/patches/openstack-no-network-config.patch
     + debian/patches/renderer-do-not-prefer-netplan.patch
-    + debian/patches/expire-on-hashed-users.patch
-  * New upstream snapshot. (LP: #1987318)
+  * New upstream snapshot. (LP: #123)
+    + DataSourceVMware: fix var use before init (#1674) [Andrew Kutz]
+    + rpm/copr: ensure RPM represents new clean.d dir artifacts (#1680)
+    + test: avoid centos leaked check of /etc/yum.repos.d/epel-testing.repo
+      (#1676)
+    + Release 22.3 (#1662)
+    + sources: obj.pkl cache should be written anyime get_data is run
+      (#1669)
+    + schema: drop release number from version file (#1664)
+    + pycloudlib: bump to quiet azure HTTP info logs (#1668)
+    + test: fix wireguard integration tests (#1666)
+    + Github is deprecating the 18.04 runner starting 12.1 (#1665)
+    + integration tests: Ensure one setup for all tests (#1661)
+    + tests: ansible test fixes (#1660)
+    + Prevent concurrency issue in test_webhook_hander.py (#1658)
+    + Workaround net_setup_link race with udev (#1655)
+    + test: drop erroneous lxd assertion, verify command succeeded (#1657)
     + Fix Chrony usage on Centos Stream (#1648)
       [Sven Haardiek]
     + sources/azure: handle network unreachable errors for saveable PPS
@@ -93,8 +107,7 @@ cloud-init (22.3-3-g9f0efc47-0ubuntu1~18.04.1) bionic; urgency=medium
     + meta-schema: add infra to skip inapplicable modules
       [Alberto Contreras]
     + sources/azure: don't set cfg["password"] for default user pw
-      (#1592)
-      [Chris Patterson]
+      (#1592) [Chris Patterson]
     + schema: activate grub-dpkg deprecations (#1600) [Alberto Contreras]
     + docs: clarify user password purposes (#1593)
     + cc_lxd: Add btrfs and lvm lxd storage options (SC-1026) (#1585)
@@ -112,8 +125,8 @@ cloud-init (22.3-3-g9f0efc47-0ubuntu1~18.04.1) bionic; urgency=medium
       [Peter Mescalchin]
     + Update govc command in VMWare walkthrough (#1576) [manioo8]
     + Update .github-cla-signers (#1588) [Daniel Mullins]
-    + Rename the openmandriva user to omv (#1575) [Bernhard
-      Rosenkraenzer]
+    + Rename the openmandriva user to omv (#1575)
+      [Bernhard Rosenkraenzer]
     + sources/azure: increase read-timeout to 60 seconds for wireserver
       (#1571) [Chris Patterson]
     + Resource leak cleanup (#1556)
@@ -131,8 +144,8 @@ cloud-init (22.3-3-g9f0efc47-0ubuntu1~18.04.1) bionic; urgency=medium
       [Alberto Contreras]
     + Schema deprecation handling (#1549) [Alberto Contreras]
     + Enable pytest to run in parallel (#1568)
-    + sources/azure: refactor ovf-env.xml parsing (#1550) [Chris
-      Patterson]
+    + sources/azure: refactor ovf-env.xml parsing (#1550)
+      [Chris Patterson]
     + schema: Force stricter validation (#1547)
     + ubuntu advantage config: http_proxy, https_proxy (#1512)
       [Fabian Lichtenegger-Lukas]
@@ -163,8 +176,7 @@ cloud-init (22.3-3-g9f0efc47-0ubuntu1~18.04.1) bionic; urgency=medium
     + Add python3-debconf to pkg-deps.json Build-Depends (#1535)
       [Alberto Contreras]
     + redhat spec: udev/rules.d lives under /usr/lib on rhel-based
-      systems
-      (#1536)
+      systems (#1536)
     + tests/azure: add test coverage for DisableSshPasswordAuthentication
       (#1534) [Chris Patterson]
     + summary: Add david-caro to the cla signers (#1527) [David Caro]
@@ -178,16 +190,14 @@ cloud-init (22.3-3-g9f0efc47-0ubuntu1~18.04.1) bionic; urgency=medium
     + CI: drop linters from Travis CI (moved to GH Actions) (#1530)
       [Paride Legovini]
     + sources/azure: remove unused encoding support for customdata
-      (#1526)
-      [Chris Patterson]
+      (#1526) [Chris Patterson]
     + sources/azure: remove unused metadata captured when parsing ovf
       (#1524) [Chris Patterson]
     + sources/azure: remove dscfg parsing from ovf-env.xml (#1522)
       [Chris Patterson]
     + Remove extra space from ec2 dual stack crawl message (#1521)
     + tests/azure: use namespaces in generated ovf-env.xml documents
-      (#1519)
-      [Chris Patterson]
+      (#1519) [Chris Patterson]
     + setup.py: adjust udev/rules default path (#1513)
       [Emanuele Giuseppe Esposito]
     + Add python3-deconf dependency (#1506) [Alberto Contreras]
@@ -196,13 +206,12 @@ cloud-init (22.3-3-g9f0efc47-0ubuntu1~18.04.1) bionic; urgency=medium
     + sources/azure: remove unused userdata property from ovf (#1516)
       [Chris Patterson]
     + sources/azure: minor refactoring to network config generation
-      (#1497)
-      [Chris Patterson]
+      (#1497) [Chris Patterson]
     + net: Implement link-local ephemeral ipv6
     + Rename function to avoid confusion (#1501)
     + Fix cc_phone_home requiring 'tries' (#1500)
     + datasources: replace networking functions with stdlib and
-  *     cloudinit.net code
+      cloudinit.net code
     + Remove xenial references (#1472) [Alberto Contreras]
     + Oracle ds changes (#1474) [Alberto Contreras]
     + improve runcmd docs (#1498)
@@ -236,22 +245,6 @@ cloud-init (22.3-3-g9f0efc47-0ubuntu1~18.04.1) bionic; urgency=medium
     + Changelog: correct errant classification of LP issues as GH (#1464)
     + Use Network-Manager and Netplan as default renderers for RHEL and
       Fedora (#1465) [Emanuele Giuseppe Esposito]
-    + DataSourceVMware: fix var use before init (#1674) [Andrew Kutz]
-    + rpm/copr: ensure RPM represents new clean.d dir artifacts (#1680)
-    + test: avoid centos leaked check of /etc/yum.repos.d/epel-testing.repo
-      (#1676)
-    + Release 22.3 (#1662)
-    + sources: obj.pkl cache should be written anyime get_data is run
-      (#1669)
-    + schema: drop release number from version file (#1664)
-    + pycloudlib: bump to quiet azure HTTP info logs (#1668)
-    + test: fix wireguard integration tests (#1666)
-    + Github is deprecating the 18.04 runner starting 12.1 (#1665)
-    + integration tests: Ensure one setup for all tests (#1661)
-    + tests: ansible test fixes (#1660)
-    + Prevent concurrency issue in test_webhook_hander.py (#1658)
-    + Workaround net_setup_link race with udev (#1655)
-    + test: drop erroneous lxd assertion, verify command succeeded (#1657)
 
  -- James Falcon <james.falcon@canonical.com>  Mon, 22 Aug 2022 14:22:42 -0500
 
```

**Comment by TheRealFalcon** [comment]
@blackboxsw  Looks good. I can apply that. Why the removal of these two lines?
`-  * d/p/renderer-do-not-prefer-netplan refresh to activators change`
`-    + debian/patches/expire-on-hashed-users.patch`

**Comment by blackboxsw** [comment]
> @blackboxsw Looks good. I can apply that. Why the removal of these two lines? `- * d/p/renderer-do-not-prefer-netplan refresh to activators change` `- + debian/patches/expire-on-hashed-users.patch`

I only thought that was actually a manually added changelog entry which duplicated the "refresh patches" messages section.  I realize it was more than that simple patch refresh, it was a 'backport' though. so maybe worth keeping.

**Review by blackboxsw — APPROVED** [review]
- build-package sbuild good, no new lintian errors
- quilt push -a; tox -e py3 | check_format; quilt pop -a GOOD
- version string appropriate for bionic, only SRU bug listed in changelog

**Pull request merged**